### PR TITLE
Add automated script to the data plane

### LINKED ISSUE #1496: Setup instructions with Workload Identity is too complicated
**Problem**
If you want to setup Knative on GKE with Knative-GCP project and with Workload Identity auth, the setup instructions are way too complicated. You need to first follow [install-knative-gcp](https://github.com/google/knative-gcp/blob/master/docs/install/install-knative-gcp.md) and then follow [dataplane-service-account.md](https://github.com/google/knative-gcp/blob/master/docs/install/dataplane-service-account.md). There are many steps, some of them are manual, some of them are script based. In the end, it's very error prone. I couldn't get it working and in the end, I reverted to Kubernetes secrets. 

**[Persona:](https://github.com/knative/eventing/blob/master/docs/personas.md)** 
Anyone wanting to read Google Cloud events into Knative.

**Exit Criteria**
Give me a single script or a controller or something that sets everything up for me with defaults.

**Time Estimate (optional):** 
2-3 days

### PR DESCRIPTION
**Pull request by grac3gao**
Fixes #1496 

## Proposed Changes

-
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: docs/install/dataplane-service-account.md [hunk 1]
```diff
@@ -90,117 +108,83 @@ Plane:
...
-1. Create a secret on the Kubernetes cluster with the downloaded key. Remember
-   to create the secret in the namespace your resources will reside. The example
-   below does so in the `default` namespace.
+  Instead of manually configuring Workload Identity namespace by namespace,
+  you can authorize the Controller to configure Workload Identity for you.
 
-   ```shell
-   kubectl --namespace default create secret generic google-cloud-key --from-file=key.json=cre-dataplane.json
-   ```
+  Apply [init_control_plane_gke.sh](../../hack/init_control_plane_gke.sh) without parameters:
```
Comment (capri-xiyue) [inline docs/install/dataplane-service-account.md:144]: This should be init_control_planeXXX.sh or init_data_planeXXX.sh?

File: docs/install/dataplane-service-account.md [hunk 2]
```diff
@@ -90,117 +108,83 @@ Plane:
...
 
-   `google-cloud-key` and `key.json` are default values expected by our
-   resources.
+  ```shell
+  ./hack/init_control_plane_gke.sh
```
Comment (capri-xiyue) [inline docs/install/dataplane-service-account.md:147]: same here.

File: docs/install/dataplane-service-account.md (hunk 1 above)
Comment (grac3gao) [reply docs/install/dataplane-service-account.md:144]: data_plane, thank you for finding it.

**Pull request merged**